.github/scripts/depexts/generate-actions.sh

```diff
@@ -102,6 +102,12 @@ RUN apt install -y $mainlibs $ocaml
 RUN apt install -y g++
 EOF
     ;;
+  nix)
+    cat >$dir/Dockerfile << EOF
+FROM nixos/nix
+RUN nix-channel --update
+RUN nix-env -i gnum4 git rsync patch bzip2 gnumake wget ocaml ocaml5.1.1-ocaml-compiler-libs unzip gcc diffutils patch getconf-glibc gnused
```

Comment:
while trying locally, i had a not found for `ocaml5.1.1-ocaml-compiler-libs`, it workd with `ocaml5.2.1-ocaml-compiler-libs`, but we can't change the script for each new version of the package. Is it possible to not have it version dependant?

Reply:
querying and use it sounds like the way to go

Reply:
```suggestion
RUN nix-env -iA nixpkgs.gnum4 nixpkgs.git nixpkgs.rsync nixpkgs.patch nixpkgs.bzip2 nixpkgs.gnumake nixpkgs.wget nixpkgs.ocaml nixpkgs.ocamlPackages.ocaml-compiler-libs nixpkgs.unzip nixpkgs.gcc nixpkgs.diffutils nixpkgs.patch nixpkgs.getconf nixpkgs.gnused
```

Reply:
Ah, magic!